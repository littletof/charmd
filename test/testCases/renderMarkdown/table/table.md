|   a   |   b   |   c   |
| ----- | :---: | ----: |
|  123  |  456  |  789  |
|  ABCDEF  |  DEFGHI  |  GHIJKL  |

| left1 |left2|   center   |  right  |
|-------|:----|:-----:|----:|
|  123  | 456 |  789  | 101 |
|  `ABC`  | **DEF** |  *GHI*  | ![image](docs/showcase.png) |

> |   a   |   b   |   c   |
> | ----- | :---: | ----: |
> |  123  |  456  |  789  |
> |  ABCDEF  |  DEFGHI  |  GHIJKL  |
