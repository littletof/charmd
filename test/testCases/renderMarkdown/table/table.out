┌────────┬────────┬────────┐
│ [34m[1ma[22m[39m      │   [34m[1mb[22m[39m    │      [34m[1mc[22m[39m │
├────────┼────────┼────────┤
│ 123    │  456   │    789 │
├────────┼────────┼────────┤
│ ABCDEF │ DEFGHI │ GHIJKL │
└────────┴────────┴────────┘

┌───────┬───────┬────────┬────────────────────────────────────┐
│ [34m[1mleft1[22m[39m │ [34m[1mleft2[22m[39m │ [34m[1mcenter[22m[39m │                              [34m[1mright[22m[39m │
├───────┼───────┼────────┼────────────────────────────────────┤
│ 123   │ 456   │  789   │                                101 │
├───────┼───────┼────────┼────────────────────────────────────┤
│ [7m ABC [27m │ [1mDEF[22m   │  [3mGHI[23m   │ [90m[3mImage: ![image](docs/showcase.png)[23m[39m │
└───────┴───────┴────────┴────────────────────────────────────┘

┃ [90m[3m┌────────┬────────┬────────┐[23m[39m
┃ [90m[3m│ [34m[1ma[22m[90m      │   [34m[1mb[22m[90m    │      [34m[1mc[22m[90m │[23m[39m
┃ [90m[3m├────────┼────────┼────────┤[23m[39m
┃ [90m[3m│ 123    │  456   │    789 │[23m[39m
┃ [90m[3m├────────┼────────┼────────┤[23m[39m
┃ [90m[3m│ ABCDEF │ DEFGHI │ GHIJKL │[23m[39m
┃ [90m[3m└────────┴────────┴────────┘[23m[39m
